src/MobileMapping/MobileMappingLayer.js

```diff
@@ -1,148 +0,0 @@
-/**
```

Comment:
I removed all Layer(s) instance.
And as MobileMapping feature was broken I didn't port it to the new arch.